virttest/env_process.py

```diff
@@ -677,14 +678,37 @@ def preprocess(test, params, env):
     if params.get("storage_type") == "nfs":
         image_nfs = nfs.Nfs(params)
         image_nfs.setup()
+        if params.get("migration_setup", "no") == "yes":
+            # Configure NFS client on remote host
+            params["server_ip"] = params.get("remote_ip")
+            params["server_user"] = params.get("remote_user", "root")
+            params["server_pwd"] = params.get("remote_pwd")
+            params["client_ip"] = params.get("local_ip")
+            params["client_user"] = params.get("local_user", "root")
+            params["client_pwd"] = params.get("local_pwd")
+            params["nfs_client_ip"] = params.get("remote_ip")
+            params["nfs_server_ip"] = params.get("local_ip")
+            nfs_client = nfs.NFSClient(params)
+            nfs_client.setup()
         distro_details = distro.detect()
         if distro_details.name.upper() != 'UBUNTU':
             if params.get('set_sebool_local', 'yes') == "yes":
                 params['set_sebool_local'] = "yes"
                 params['local_boolean_varible'] = "virt_use_nfs"
                 params['local_boolean_value'] = "on"
-                seLinuxBool = utils_misc.SELinuxBoolean(params)
-                seLinuxBool.setup()
+        # configure selinux on remote host to permit migration
+        if params.get("migration_setup", "no") == "yes":
+            cmd = "cat /etc/os-release | grep '^PRETTY_NAME'"
+            session = test_setup.remote_session(params)
+            if 'UBUNTU' not in str(session.cmd_output(cmd)).upper():
+                params["set_sebool_remote"] = "yes"
+                params["remote_boolean_varible"] = "virt_use_nfs"
+                params["remote_boolean_value"] = "on"
+        if(params.get('set_sebool_local') == "yes" or
```

Comment:
params.get('set_sebool_local') == "yes"  and params.get('set_sebool_remote') == "yes" were called multiple times,suggest define them as variables at the beginning.